### PR TITLE
UHM-5124, migrate only obs with form_version 3

### PR DESCRIPTION
**Pull request by cioan**

### CODE REVIEW

File: etl/src/main/java/org/pih/hivmigration/etl/sql/HivExamOisMigrator.groovy [hunk 1]
```diff
@@ -99,15 +102,29 @@ class HivExamOisMigrator extends ObsMigrator {
...
+                end as form_name,
+                case
+                    when (x.encounter_id in (select k.encounter_id from HIV_INTAKE_FORMS k where k.encounter_id=x.encounter_id)) then 
+                    ( select form_version from HIV_INTAKE_FORMS where encounter_id=x.encounter_id ) 
+                    when (x.encounter_id in (select f.encounter_id from HIV_FOLLOWUP_FORMS f where f.encounter_id=x.encounter_id)) then 
+                    ( select form_version from HIV_FOLLOWUP_FORMS where encounter_id=x.encounter_id ) 
+                    else null 
+                end as form_version
             from HIV_EXAM_OIS x, HIV_ENCOUNTERS e, hiv_demographics_real d  
             where x.ENCOUNTER_ID = e.ENCOUNTER_ID and e.patient_id = d.patient_id;
```
Comment (brandones) [inline etl/src/main/java/org/pih/hivmigration/etl/sql/HivExamOisMigrator.groovy:129]: Any reason you didn't just `JOIN` on the `_FORMS` tables? Would simplify the code and might be faster.

**Review by brandones — APPROVED** [review]
Looks generally good to me, just one suggestion/note

File: etl/src/main/java/org/pih/hivmigration/etl/sql/HivExamOisMigrator.groovy (hunk 1 above)
Comment (mogoodrich) [reply etl/src/main/java/org/pih/hivmigration/etl/sql/HivExamOisMigrator.groovy:129]: Yeah, I think a join would work here?

Comment (cioan) [reply etl/src/main/java/org/pih/hivmigration/etl/sql/HivExamOisMigrator.groovy]: Thanks guys! I have updated the query to use joins. Thanks!

**Pull request merged**